### PR TITLE
integration test working

### PR DESCRIPTION
**Pull request by ineiti**
Made the integration-test working again, by changing:
- taking new `Makefile` and `install.sh` from `cothority`
- replacing `codegangsta/cli` with `urfave/cli` (this is the new version1)
- making `cosi check` fail with `exit(1)`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,44 +1,35 @@
...
 
 test_lint:
 	@echo Checking linting of files
 	@{ \
-	go get -u github.com/golang/lint/golint; \
-	exclude="_test.go"; \
-	lintfiles=$$( golint ./... | egrep -v "($$exclude)" ); \
-	if [ -n "$$lintfiles" ]; then \
+		go get -u github.com/golang/lint/golint; \
+		exclude="protocols/byzcoin|_test.go"; \
```
Comment (nikkolasg) [inline Makefile:18]: why do we exclude byzcoin from the linter ? I know it's been here for quite some time but only now I'm asking why ... :)

File: client.go [hunk 2]
```diff
@@ -32,58 +32,8 @@ func checkConfig(c *cli.Context) error {
...
-		err := verifySignatureHash([]byte(msg), sig, list)
-		if err != nil {
-			fmt.Fprintln(os.Stderr,
-				fmt.Sprintf("Invalid signature: %v", err))
-		} else {
-			fmt.Println("Success")
-		}
-	}
-
+	err = server.CheckServers(group)
```
Comment (nikkolasg) [inline client.go:35]: +1

File: Makefile (hunk 1 above)
Comment (ineiti) [reply Makefile:18]:
```
$ golint ./... | wc
      76     831    7699
```

ByzCoin is not in very good shape. Linus T. would use more colorful language for that ;)

Comment (liamsi) [reply Makefile:18]: I think it always was excluded, no? We (very) hastily written that code and never cared/had the time to really polish it (and @LefKok could do without that).

**Comment by nikkolasg** [comment]
OK for me.

File: Makefile (hunk 1 above)
Comment (liamsi) [reply Makefile:18]: > Linus T. would use more colorful language for that ;)

I think Linus T. would explode in colorful rainbow of rants if he saw (large parts of) our code (not only byzcoin) 🌈 💥 😄

**Pull request merged**